src/components/FileTreeNode/index.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { TreefoldRenderProps } from 'react-treefold';
+import { ListGroup } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import makeClassName from 'classnames';
+
+import { gettext } from '../../utils';
+import { TreeNode } from '../FileTree';
+import styles from './styles.module.scss';
+
+type PublicProps = TreefoldRenderProps<TreeNode>;
+
+class FileTreeNodeBase<TreeNodeType> extends React.Component<PublicProps> {
+  render() {
+    const {
+      node,
+      level,
+      isFolder,
+      isExpanded,
+      getToggleProps,
+      hasChildNodes,
+      renderChildNodes,
+    } = this.props;
+
+    let listGroupItemProps = {
+      className: makeClassName(styles.node, {
+        [styles.directoryNode]: isFolder,
```

Comment:
Why do we need this? It looks like it's just for testing, but it also looks like the tests could confirm the same thing based on the icon, and don't really need to use this class just to prove that a node is a directory node.

Reply:
If we get rid of the icon or change it, tests should be updated. It's less probable with a CSS class, no? Also, I guess we'll end up styling those directories a bit in the near future :p